integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/chat_generator.py

```diff
@@ -49,7 +49,7 @@ class AmazonBedrockChatGenerator:
 
     SUPPORTED_MODEL_PATTERNS: ClassVar[Dict[str, Type[BedrockModelChatAdapter]]] = {
         r"anthropic.claude.*": AnthropicClaudeChatAdapter,
-        r"meta.llama2.*": MetaLlama2ChatAdapter,
+        r"meta.llama.*": MetaLlama2ChatAdapter,
```

Comment:
Hey @tstadel I don't think we should do this. The `MetaLlama2ChatAdapter` includes with it the correct prompt template structure with the correct instruction tokens for Llama2. And the instruction tokens for Llama3 do not match. So for the ChatAdapater we'd need to make one specific to Llama3.

Reply:
Ah ok, got it. Thought it would work the same as the non-chat version. I'll revert.